e2e/tests/odoo-openmrs-flows.spec.ts

```diff
@@ -227,6 +227,24 @@ test('Discontinuing a synced OpenMRS lab order for an Odoo customer with a singl
   await expect(page.locator('tr.o_data_row:nth-child(1) td:nth-child(8) span')).toHaveText('Cancelled');
 });
 
+test('Ordering a lab test for an OpenMRS patient with weight creates the corresponding Odoo customer, with a quotation that includes the weight.', async ({ page }) => {
```

Comment:
We need one more test which should cover the case:
Lab test/ Drug order is created on OpenMRS and synced in Odoo. Odoo quotation will not have Patient weight synced, now Patient Vitals are recorded and a new item is ordered in OpenMR then Odoo quotation should have Patient weight synced.